docs/data/index.html

```diff
@@ -0,0 +1,333 @@
+---
+title: Test Suite Stats
+layout: common
+---
+<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/chartjs-plugin-trendline"></script>
+
```

Comment:
Using these to chart the data.  We could also use the Google Charts for charting, but these charts seemed nicer/more flexible.  So I opted for charts.js.